etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/results/SendResultUseCase.java

```diff
@@ -0,0 +1,24 @@
+package gov.hhs.cdc.trustedintermediary.etor.results;
+
+import gov.hhs.cdc.trustedintermediary.etor.messages.MessageSender;
+import gov.hhs.cdc.trustedintermediary.etor.messages.SendMessageUseCase;
+import gov.hhs.cdc.trustedintermediary.etor.messages.UnableToSendMessageException;
+import javax.inject.Inject;
+
+/** Use case for converting and sending a lab result message. */
+public class SendResultUseCase implements SendMessageUseCase<Result<?>> {
+    private static final SendResultUseCase INSTANCE = new SendResultUseCase();
+    @Inject MessageSender<Result<?>> sender;
+
+    private SendResultUseCase() {}
+
+    public static SendResultUseCase getInstance() {
+        return INSTANCE;
+    }
+
+    @Override
+    public void convertAndSend(Result<?> result, String receivedSubmissionId)
```

Comment:
Should the conversion happen in the use case or should that be a separate class to handle that?

Reply:
I think it should. That's what we're doing in `SendOrderUseCase`. Maybe the use case should be named `ConvertAndSendResultUseCase` but I don't know if that's necessary